src/main/webapp/config/oauth.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
same as above . . .just need to use "sample_oauth.xml" if not present then don't have